msteams-platform/apps-in-teams-meetings/API-references.md

```diff
@@ -345,7 +345,7 @@ Use the following example to configure your app manifest's `webApplicationInfo`
 >
 > * The bot can receive meeting start or end events automatically from all the meetings created in all the channels by adding `ChannelMeeting.ReadBasic.Group` to manifest for RSC permission.
 >
-> * For one-on-one and group calls `organizer` is the initiator of the chat.
+> * For a one-on-one call `organizer` is the initiator of the chat and for group calls `organizer` is the call initiator.
```

Comment:
> For a one-on-one call `organizer` is the initiator of the chat

Who's the initiator of a chat-- the person that sent the first message? Does this mean that if A starts a chat with B, and then B starts a call by pressing the call button, the API will report the organizer as A, not B?

Reply:
@leonyang12 But for group calls we can do that? Just so it's clear:
- if A starts a group chat with members A,B,C, then C starts a call from the group chat, then C is reported as the organizer
- if A start a 1:1 chat with B, then B starts a call from the 1:1 chat, then A is reported as the organizer